server_easyjson.go

```diff
@@ -75,19 +134,16 @@ func easyjson22b57fa5DecodeGithubComGoOpenapiSpec3(in *jlexer.Lexer, out *Server
 		in.Consumed()
 	}
 }
-func easyjson22b57fa5EncodeGithubComGoOpenapiSpec3(out *jwriter.Writer, in ServerVariable) {
+func easyjson22b57fa5EncodeGithubComGoOpenapiSpec31(out *jwriter.Writer, in ServerVariable) {
 	out.RawByte('{')
 	first := true
 	_ = first
-	if len(in.Enum) != 0 {
-		const prefix string = ",\"enum\":"
-		if first {
-			first = false
-			out.RawString(prefix[1:])
+	{
+		const prefix string = ",\"Enum\":"
```

Comment:
The casing has changed, is that on purpose?

Reply:
No, it wasn't on purpose. I had to generate the easyjson file again once the structs in `server.go` got updated. And, I didn't use the `-lower_camel_case` argument while generating the easyjson file. That's why they were in CamelCase. Again, it's my fault also that I didn't check the difference of the generated files assuming they would be correct. Will be fixing it in 12 hrs.

Reply:
Generated the server and contact easyjson files correctly. And, also found your `go:generate` commands in `docs.go`. I wish I had found them earilier.